ngx-tools/testing/src/mocks/retry-with-escalation.mock.ts

```diff
@@ -72,5 +74,6 @@ export class RetryWithEscalationMock<CM = ClaimMap> extends RetryWithEscalation
 
 
```

Comment:
Can you clarify what you mean?

Reply:
Reading a later comment I think I understand..

JSDoc comments about deprecation go on the item that is deprecated. This is simply an internal comment for our own linter because we are using a deprecated item.